docs/json_rpc_api.md

```diff
@@ -1166,24 +1168,25 @@ the notifications.
 package main
 
 import (
-	"github.com/decred/dcrrpcclient"
-	"github.com/decred/dcrd/dcrutil"
-	"github.com/decred/dcrd/wire"
 	"io/ioutil"
 	"log"
 	"path/filepath"
 	"time"
+
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/decred/dcrd/rpcclient"
 )
 
 func main() {
 	// Setup handlers for blockconnected and blockdisconnected
 	// notifications.
-	ntfnHandlers := dcrrpcclient.NotificationHandlers{
-		OnBlockConnected: func(hash *wire.ShaHash, height int32) {
-			log.Printf("Block connected: %v (%d)", hash, height)
+	ntfnHandlers := rpcclient.NotificationHandlers{
+		OnBlockConnected: func(blockHeader []byte, transactions [][]byte) {
+			log.Printf("Block connected: %v %v", blockHeader, transactions)
```

Comment:
Would you mind changing these to `%x` and updating the sample output accordingly?   This is leftover when when they used to since the hashes instead of the actual data.

Reply:
Sure, makes sense.